packages/flutter_tools/doc/daemon.md

```diff
@@ -250,6 +250,7 @@ See the [source](https://github.com/flutter/flutter/blob/master/packages/flutter
 
 ## Changelog
 
+- 0.5.1: Added `platformType`, `ephemeral`, and `category` field to device.
```

Comment:
We should also update the docs for the `device.getDevices` call and the `device.added` and `device.removed` events.